### ISSUE TITLE
Bad performance on iWILDs

### ISSUE THREAD

**Issue by LixDemon** [locuslab/FLYP#12]
I run your code and got a much worse result with the paper on iWILDcam. The performance is inconsistent either on acc or f1-macro. Have you got any idea why this happened?

**Comment by SachinG007** [comment]
@LixDemon , did you check the hyper-parameters in the paper. This shouldn't really happen as the numbers in our paper are over 5 seeds.

**Comment by LixDemon** [comment]
I utilized the hyperparameters in the readme file. I guess the numbers in the paper should be the macro-f1 score rather than top-1 accuracy.

**Comment by SachinG007** [comment]
Yes, that' correct. We use the metrics used to compare in the literature (which is F1 for WILDS). Will clarify this in the table.
